_posts/2020-02-26-flink-for-data-warehouse.md

```diff
@@ -0,0 +1,107 @@
+
+Secondly, the infrastructure should be able to handle both offline batch data for offline analytics and exploration, and online streaming data for more timely analytics. Both are indispensable as they both have very valid use cases. Despite real time processing mentioned above, batch would still exist as it’s good for ad hoc queries and explorations, and full size calculations. Your modern infrastructure should not force users to choose between one or the other, it should offer users both options for a world-class data infrastructure.
+
+Thirdly, the data players, including data engineers, data scientists, analysts, and operations, urge a more unified infrastructure than ever before for easier ramp-up and higher working efficiency. The big data landscape has been fragmented for years, companies may have one set of infrastructure for real time processing, one set for batch, one set for OLAP, etc. That may came from the legacy of lambda architecture, which was popular in the era when stream processors weren't mature and users have to use periodical batch as correct to streaming pipeline. Well, it's a different era now! As stream processing is becoming main stream and dominant, end users no longer want to learn shattered pieces of skills and maintain too many moving parts with all kinds of tools and pipelines. What they really need is to have a unified analytics platform that can be learned and mastered more easily, and simplify any operation complexity.
+
+If any of these resonate with you, you jsut found the right post to read: We have never been this close to the vision by strengthening Flink’s integration with Hive to a production grade.
+
+
+## Flink and Its Integration With Hive Comes into the Scene
+
+Apache Flink has been a proven scalable system to handle extremely high workload of streaming data in super low latency in many giant tech companies.
+
+Despite its huge success in real time processing domain, at its deep root, Flink has been faithfully following its inborn philosophy of being [an unified data processing engine for both batch and streaming](https://flink.apache.org/news/2019/02/13/unified-batch-streaming-blink.html), and taking a streaming-first approach in its architecture to do batch processing. By making batch a special case for streaming, Flink really leverages its cutting edge streaming capabilites and applies them to batch scenarios to gain the best offline performance. Flink’s batch performance has been quite outstanding in the early days, and has become even more impressive, as the community began to merge Blink, an Alibaba’s fork of Flink, back to Flink in 1.9 and finished it in 1.10, according to benchmarks we ran. 
```

Comment:
"According to benchmarks we ran". I would refrain from mentioning benchmarks here if we are not supporting it with some numbers, charts or links to publicly available benchmark scripts.